elastalert/config.py

```diff
@@ -108,12 +108,15 @@ def load_options(rule, conf=None):
     rule.setdefault('realert', datetime.timedelta(seconds=0))
     rule.setdefault('aggregation', datetime.timedelta(seconds=0))
     rule.setdefault('query_delay', datetime.timedelta(seconds=0))
-    rule.setdefault('timestamp_field', '@timestamp')
+    rule.setdefault('timestamp_field' , rule.get('timestamp_field' , '@timestamp'))
```

Comment:
This change doesn't do anything. setdefault doesn't have any effect if timestamp_field is already in the dictionary.